admin/src/main/scala/io/buoyant/admin/StaticFilter.scala

```diff
@@ -0,0 +1,30 @@
+package io.buoyant.admin
+
+import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle.http.{MediaType, Response, Request}
+import com.twitter.util.Future
+
+object StaticFilter extends SimpleFilter[Request, Response] {
+
+  private[this] val contentTypes: Map[String, String] = Map(
+    "css" -> "text/css",
+    "gif" -> MediaType.Gif,
+    "html" -> MediaType.Html,
+    "jpeg" -> MediaType.Jpeg,
+    "jpg" -> MediaType.Jpeg,
+    "js" -> MediaType.Javascript,
+    "json" -> MediaType.Json,
+    "png" -> MediaType.Png,
+    "svg" -> "image/svg+xml",
+    "txt" -> MediaType.Txt
+  )
+
+  def apply(req: Request, svc: Service[Request, Response]) = {
+    svc(req).flatMap { res =>
+      val ext = req.path.split('.').lastOption.getOrElse("")
```

Comment:
I don't think this is correct - `"/foo/bar".split(".").lastOption` is `Some("/foo/bar")`. Can you switch this to use `req.fileExtension`?